src/Particular.PlatformSample/PlatformLauncher.cs

```diff
@@ -88,7 +88,6 @@ public static void Launch(bool showPlatformToolConsoleOutput = false, string ser
                     var serviceControlApiUrl = $"http://localhost:{controlPort}/api";
                     Console.WriteLine();
                     Console.WriteLine($"ServiceControl API can now be accessed via: {serviceControlApiUrl}");
-                    
```

Comment:
nitpick: Can you add this line back so there the is no diff in this file?

Reply:
Sure can!